### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by vibraphone** [Kitware/kw-doc#1]
Is there a way to override the logo text? In the generated HTML, the logo appears like so:

```html
    <h1 id="logo-wrap">
      <a href="/sensei/" id="logo">ParaViewWeb</a>
    </h1>
```

but my project name is not `ParaViewWeb`.

**Comment by jourdain** [comment]
Keep adding those issue so I can tackle them all at once...

**Comment by jourdain** [comment]
Fixed in 1.1.5
